biostructbenchmark/cli.py

```diff
@@ -6,23 +6,27 @@
 
 
 def validate_file_path(file_path: str) -> str:
```

Comment:
can this return type `Path` ? (add `from pathlib import Path`)
I feel that `Path` must have a clean/standard way to validate itself, which could simplify this function